src/components/forms/FieldWrapper.tsx

```diff
@@ -126,26 +135,41 @@ function FieldWrapper(props, ref) {
       aria-label={label}
       {...labelProps}
     >
-      {extra ? (
-        <Grid placeContent="baseline space-between" flow="column">
-          <div>{label}</div>
-          <Text preset="t3">{extra}</Text>
-        </Grid>
-      ) : (
-        label
-      )}
-      {tooltip ? (
-        <>
-          &nbsp;
-          <TooltipProvider
-            title={tooltip}
-            activeWrap
-            width="initial max-content 40x"
+      <Flex flex="1 0 100%">
+        {extra ? (
+          <Grid
+            placeContent="baseline space-between"
+            flow="column"
+            width="100%"
           >
-            <InfoCircleOutlined style={{ color: 'var(--primary-color)' }} />
-          </TooltipProvider>
-        </>
-      ) : null}
+            <Space placeItems="baseline" gap="0.5x">
+              <div>{labelPrefix}</div>
+              <div>{label}</div>
+              <div>{labelSuffix}</div>
+            </Space>
+
+            <Text preset="t3">{extra}</Text>
+          </Grid>
+        ) : (
+          <Space placeItems="baseline" gap="0.5x">
```

Comment:
There is no need to introduce an additional element if `suffix` is no presented.

Reply:
I simplified the label area in the last commit. However, I still doubt the idea of removing __unnecessary__ wrapper around the label since it adds additional logic without any functional and/or non-functional impact. Can you explain the reason behind your advice?

Reply:
I don't like to see a mess in markup. But in the case of additional wrappers, it almost always better to reduce them so you have better control over their position.
Additional wrappers can also create unnecessary regions for readers which are not intended.

Reply:
Not sure it's applicable to our situation